yaml-mode.el

```diff
@@ -178,7 +178,7 @@ that key is pressed to begin a block literal."
              "y" "Y" "yes" "Yes" "YES" "n" "N" "no" "No" "NO"
              "true" "True" "TRUE" "false" "False" "FALSE"
              "on" "On" "ON" "off" "Off" "OFF") t)
-          " *$")
+          "\\b")
```

Comment:
I've tested some more meanwhile and it does match `True#123` which it shouldn't. So `\\_>` is better.